app/assets/javascripts/3dbio_viewer/src/webapp/training-app/components/action-button/ActionButton.tsx

```diff
@@ -26,12 +26,12 @@ const StyledFab = styled(Fab)`
     align-items: center;
```

Comment:
General comment about the training app styles. At zoom 90% it looks very nice, but when I start to increase the zoom, it looks a bit off. For example, this how I see the training app at 100% with Chrome (similar in FF):

![image](https://user-images.githubusercontent.com/24643/105692007-380cd000-5efe-11eb-9e93-60c49134025a.png)

At 150%:

![image](https://user-images.githubusercontent.com/24643/105692173-6094ca00-5efe-11eb-97ed-8c639043927d.png)